bindings/matrix-sdk-ffi/src/client.rs

```diff
@@ -258,6 +286,40 @@ impl Client {
             Ok(Arc::new(session_verification_controller))
         })
     }
+
+    /// Log out the current user
+    pub fn logout(&self) -> anyhow::Result<()> {
+        RUNTIME.block_on(async move {
```

Comment:
Also, topical question: shouldn't we set `soft_logout = true` here on success? We are logged out, right? Or hasn't seen been `soft`?

Reply:
No, this logout is not _soft_.